azure_functions_worker/bindings/meta.py

```diff
@@ -94,5 +104,39 @@ def to_outgoing_proto(binding: str, obj: typing.Any, *,
             f'unable to encode outgoing TypedData: '
             f'unsupported type "{binding}" for '
             f'Python type "{type(obj).__name__}"')
+    return datum
+
 
+def to_outgoing_proto(binding: str, obj: typing.Any, *,
+                      pytype: typing.Optional[type]) -> protos.TypedData:
+    datum = get_datum(binding, obj, pytype)
     return datumdef.datum_as_proto(datum)
+
+
+def to_outgoing_param_binding(binding: str, obj: typing.Any, *,
+                      pytype: typing.Optional[type],
+                      out_name: str,
+                      shmem_mgr: SharedMemoryManager) -> protos.ParameterBinding:
+    datum = get_datum(binding, obj, pytype)
+    # TODO gochaudh: IMPORTANT: Right now we set the AppSetting to disable this
```

Comment:
Should we also check the handshake where it advertises the capabilities?

Reply:
Not sure I understand - explain?

Reply:
Right now from C# we check the capabilities of the python worker; which is the case we would expect while rollout.
I know is unlikely but do we handle the host C# side not supporting shared memory but Python yes?